lib/devise/models/database_authenticatable.rb

```diff
@@ -37,11 +37,12 @@ def self.required_fields(klass)
       # the hashed password.
       def password=(new_password)
         @password = new_password
-        self.encrypted_password = password_digest(@password) if @password.present?
+        self.encrypted_password = password_digest(@password) 
```

Comment:
@lucasmazza Most of the test cases has been failed because of I have removed `@password.present?` condition. I couldn't find the reason that why it is failing. Could you help me?